docs/engineering-defaults.md

```diff
@@ -1,7 +1,147 @@
+> day. Each integration can then be verified by an automated build and automated
+> tests.[^ci]
+
+CI is a critical part of the development lifecycle and is a natural compliment
+to [Small Batch Delivery](#small-batch-delivery). This provides developers with
+very frequent feedback on their code changes. When problematic merges occur,
+teams are quickly notified. More importantly, a bad merge can be identified
+immediately, allowing the relevant team members to quickly fix the issue.
+
+### Continuous Delivery
+
+Continuous Delivery can be traced back to the first principal of the Agile
+Manifesto[^agilemanifesto] and naturally follows Continuous Integration. It is
+the practice of sequentially deploying every viable version of an application,
+ideally to production. This ensures that the software can always be released at
+any time.
+
+### Speed
+
+As teams continually rely on CI/CD, the pipelines often become a bottleneck for
+continuing development. This is amplified by the combination of frequent code
+merges and CI/CD.
+
+A fast CI/CD pipeline provides developers with fast feedback. A slower pipeline
+adds a wasteful delay to the developer's workflow before receiving feedback. A
+very slow pipeline motivates context switching as developers perform
+other tasks while waiting for feedback. This is very expensive in terms of time
+and cognitive load.
+
+## Build for Production
+
+Teams that build for production ensure that all the work they are building is
+observable and supportable in production. These expectations are part of the
+acceptance criteria for all stories. Logs are generated, traces are stitched,
+and metrics are gathered from the beginning and updated with each change as
+needed. The team partners with Product to know what is expected from the system.
+SLIs and SLOs are defined, dashboards created, and alerting configured. The team
+knows what to do when there is an alert.
+
+It’s much harder to add observability and supportability ‘hooks’ after a feature
+is complete and usually requires significant refactoring. Putting a minimum set
+of observability and runbooks/KB in place before shipping to production
+increases resilience of the system, and allows for faster feedback.
+
+The 2022 State of DevOps report found "when a service is unreliable, users won’t
+benefit from pushing code faster into that fragile context." This means that
+benefits of the other Sensible Defaults can be limited (or unlocked) by how the
+team is Building for Production.
+
+## Build In Security
+
+Teams should always be concerned with security and this should affect how they
+plan work items. If a work item has security implications or concerns, the same
+acceptance criteria should address them. This prevents teams from separating
+security needs from other work and denies the opportunity to delay the
+implementation of security requirements.
+
+While this list is not comprehensive, it is important that:
```

Comment:
I'm hesitant to include the list. There's SO MUCH to security.

Reply:
Ya I had similar feelings. That's roughly a copy/paste from the google doc

Reply:
I think let's keep the list, but change the intro sentence to something like:

```suggestion
Security has many principles and rules that should be followed.
For example, NIST has several publications, include 800-53[^nist]
that dive deep into information technology security. The following
list is a starting point of a few practices and principles a team can begin
to follow without boiling the ocean.
```

The NIST reference is: https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53r5.pdf

Reply:
Updated.